release/long_running_tests/workloads/many_ppo.py

```diff
@@ -34,13 +35,12 @@
             "env": "CartPole-v0",
             "num_samples": 10000,
             "config": {
-                "framework": "torch",
-                "num_workers": 7,
+                "num_env_runners": 7,
                 "num_gpus": 0,
                 "num_sgd_iter": 1,
             },
             "stop": {
-                "timesteps_total": 1,
+                NUM_ENV_STEPS_SAMPLED_LIFETIME: 1,
```

Comment:
Does this mean, we only make quick tests if the class can be built and the algorithm runs?

Reply:
Yeah, I guess 🤷 . I think this is a conceptual test for tune to be able to handle large amounts of samples, and not a stress test.

Reply:
Thanks for clarifying this :)